### ISSUE TITLE
Round elements seem to be horizontally off centre

### ISSUE THREAD

**Issue by chipit24** [ProcessMaker/modeler#690]
Round elements seem to be horizontally off centre, causing the initial flow element to be not straight

**Comment by chipit24** [comment]
@velkymx Here is what it looks like when I connect sequence flows to/from start and end events in Chrome 77 on maxOS Mojave:

<img width="778" alt="Screen Shot 2019-09-25 at 4 43 11 PM" src="https://user-images.githubusercontent.com/7561061/65638195-9432ad80-dfb3-11e9-8305-f18d0273e5d5.png">

Nothing appears to be off centre or jagged. Can you provide more details on what the problem is?

**Comment by chipit24** [comment]
Try Chrome on Windows.

**Comment by chipit24** [comment]
I was able to reproduce this both on Chrome on my MacBook and on Edge in Windows 10:

<img width="156" alt="Screen Shot 2019-10-02 at 2 42 07 PM" src="https://user-images.githubusercontent.com/7561061/66072387-ee3af200-e522-11e9-88bc-f7b5fe119687.png">

It's a bit tricky as sometimes it does draw a perfectly straight line, so these steps may need to be repeated more than once:
1. Drag a task onto the grid; do not move it after dropping it.
2. Drag an end event either to the bottom or the right of the task.
3. Connect the task to the end event with a sequence flow.
4. Notice that the sequence flow is not perfectly straight, and moving the end event around to try and make the sequence flow perfectly straight does not fix the issue.
5. This may be repeated with other elements but I only tried with task and end event.

**Video of issue:**
https://www.dropbox.com/s/nubtorhfh1f8upo/sequence_flows_not_straight.mov?dl=0
(Notice that moving the task fixes the issue)

**Some thoughts:**
The paper grid size is 10px and all elements are suppose to snap to the grid to remain aligned. It's possible that after dragging / creating any of the elements, they do not correctly snap to the grid.

The app still exports two points, even though there appears to be an extra vertex added because of the bend in the flow, and Camunda renders the flow with a slight slant; so the issue appears to be because the centre of the task does not line up perfectly with the centre of the end event.
